client/public/index.html

```diff
@@ -24,6 +27,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
```

Comment:
Jenny created a branch that added this stuff and it's been merged in to `develop`. All you need to do is pull `develop` and merge it into your branch.